### PR TITLE
CE Performance Report

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: app/models/grda_warehouse/config.rb [hunk 1]
```diff
@@ -280,5 +281,28 @@ def self.arbiter_class
       # FIXME: for now, just return the one known one
       ClientAccessControl::EnrollmentArbiter if RailsDrivers.loaded.include?(:client_access_control)
     end
+
+    def self.active_supplmental_enrollment_importer_class
```
Comment (awisema) [inline app/models/grda_warehouse/config.rb]: Typo: suppl **e** mental (appears in several names)

File: drivers/ce_performance/app/models/ce_performance/results/clients_screened.rb [hunk 2]
```diff
@@ -0,0 +1,115 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module CePerformance
+  class Results::ClientsScreened < CePerformance::Result
+    include CePerformance::Results::Calculations
+    # For anyone served by CE, how long have they been in the project
```
Comment (awisema) [inline drivers/ce_performance/app/models/ce_performance/results/clients_screened.rb]:
```suggestion
    # Percentage of HoH's served in period with screen
```

File: drivers/ce_performance/app/models/ce_performance/results/event_type.rb [hunk 3]
```diff
@@ -80,30 +100,30 @@ def percentage?
     end
 
     def data_for_chart(report, comparison)
-      report_year_data = report.results.where.not(event_type: nil).pluck(:event_type, :value).to_h
-      comparison_year_data = if false # comparison.present?
-        comparison.where.not(event_type: nil).pluck(:event_type, :value).to_h
-      else
-        {}
-      end
+      # scope = report.results.where.not(event_type: nil)
```
Comment (awisema) [inline drivers/ce_performance/app/models/ce_performance/results/event_type.rb]: Remove?

File: drivers/supplemental_enrollment_data/README.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+## SupplementalEnrollmentData README
+
+This README file should be used to explain the functionality of the driver.
```
Comment (awisema) [inline drivers/supplemental_enrollment_data/README.md]: Is this driver intended to incorporate ALL the supllemental enrollment data models, or just the TPC one? I'd prefer that it be just the TPC one, in which case, should it be renamed, and described here?

File: drivers/supplemental_enrollment_data/config/routes.rb [hunk 5]
```diff
@@ -0,0 +1,6 @@
+BostonHmis::Application.routes.draw do
+  scope :supplemental_enrollment_datum do
+    # TODO
+    # get '/my_path', to: 'supplemental_enrollment_data/my_controller'
+  end
```
Comment (awisema) [inline drivers/supplemental_enrollment_data/config/routes.rb]: Do empty route scopes affect the app at all?

**Review by awisema — CHANGES_REQUESTED** [review]

File: drivers/supplemental_enrollment_data/config/routes.rb (hunk 5 above)
Comment (eanders) [reply drivers/supplemental_enrollment_data/config/routes.rb]: they shouldn't, but I'll pull that.

**Pull request merged**